### PR TITLE
[extractor] added Jeniusplay

### PR DESCRIPTION
**Pull request by hexated**

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/extractors/Jeniusplay.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        )
+                    )
+                }
+            }
+        }
+    }
+
+    private fun getLanguage(str: String): String {
+        return when {
+            str.lowercase().contains("indonesia") || str.lowercase()
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/extractors/Jeniusplay.kt:55]: .contains("indonesia", ignoreCase = true) is better imo

File: app/src/main/java/com/lagradost/cloudstream3/extractors/Jeniusplay.kt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+        document.select("script").map { script ->
+            if (script.data().contains("eval(function(p,a,c,k,e,d)")) {
+                val subData =
+                    getAndUnpack(script.data()).substringAfter("\"tracks\":[").substringBefore("],")
+                tryParseJson<List<Tracks>>("[$subData]")?.map { subtitle ->
+                    subtitleCallback.invoke(
+                        SubtitleFile(
+                            getLanguage(subtitle.label.toString()),
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/extractors/Jeniusplay.kt:44]: This will be "null" if there is no label, perhaps a fallback?

**Review by Blatzar — APPROVED** [review]
Looks good, will probably merge tomorrow if that's okay :+1:

**Comment by hexated** [comment]
Yeah.. 👍

**Pull request merged**